docs/releasing.md

```diff
@@ -96,6 +96,12 @@ machine and update the release branch:
 git fetch && git reset --hard $LK_REMOTE/rel_$LK_RELEASE
 ```
 
+Finally, with all of the arch-specific images out, push out the multi-arch index:
```

Comment:
I think with this, the comment above regarding that this must be done on `x86_64` first is no longer true. The only requirement is to build this in turns...you can even build in parallel and copy the `versions.<arch>` from the build machines

Reply:
Good point. I updated.